Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -19,13 +19,13 @@ In order to implement an instrument you need to implement the `Instrument` proto
 
 `Instrument` has two requirements:
 
-1. An `Instrument/extract(_:into:using:)` method, which extracts values from a generic carrier (e.g. HTTP headers) and store them into a `Baggage` instance
-2. An `Instrument/inject(_:into:using:)` method, which takes values from the `Baggage` to inject them into a generic carrier (e.g. HTTP headers)
+1. An `Instrument/extract(_:into:using:)` method, which extracts values from a generic carrier (e.g. HTTP headers) and store them into a `ServiceContext` instance
+2. An `Instrument/inject(_:into:using:)` method, which takes values from the `ServiceContext` to inject them into a generic carrier (e.g. HTTP headers)
 
 The two methods will be called by instrumented libraries/frameworks at asynchronous boundaries, giving you a chance to
 act on the provided information or to add additional information to be carried across these boundaries.
 
-> The [`Baggage` documentation](https://github.com/apple/swift-distributed-tracing-baggage) type is declared in the swift-distributed-tracing-baggage package.
+> The [`ServiceContext` documentation](https://github.com/apple/swift-service-context) type is declared in the swift-service-context package.
```

Comment:
Same here, let’s point to the documentation instead of source repository.

Reply:
That's a good idea, done